script/bootstrap-on-linux.sh

```diff
@@ -0,0 +1,1162 @@
+#!/usr/bin/env bash
+#====================================
+# @file   : bootstrap-on-linux.sh
+# @brief  : installs dependencies for building Vega Strike on Linux
+# @usage  : sudo script/bootstrap-on-linux.sh 1 (to update all installed
+#           packages on the system in the process)
+#     or  : sudo script/bootstrap-on-linux.sh 0 (to skip updating, and
+#           just install the new packages that are needed)
+# @param  : just one parameter, either a 1 or a 0, to indicate whether or not to
+#           UPDATE_ALL_SYSTEM_PACKAGES
+#====================================
+# Copyright (C) 2020-2025 Roy Falk, Stephen G. Tuggy, Benjamen R. Meyer,
+# and other Vega Strike contributors
+#
+# This file is part of Vega Strike.
+#
+# Vega Strike is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Vega Strike is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Vega Strike. If not, see <https://www.gnu.org/licenses/>.
+
+set -e
+
+echo "------------------------------------------"
+echo "--- bootstrap-on-linux.sh | 2025-11-27 ---"
+echo "------------------------------------------"
+
+UPDATE_ALL_SYSTEM_PACKAGES="$1"
+
+if [ -f /etc/os-release ]
+then
+    OS_RELEASE_LOCATION="/etc/os-release"
+elif [ -f /usr/lib/os-release ]
+then
+    OS_RELEASE_LOCATION="/usr/lib/os-release"
+else
+    echo "os-release file not found; unable to continue"
+    exit 1
+fi
+LINUX_ID=$(grep ^ID= $OS_RELEASE_LOCATION | sed 's/^ID=//' | tr -d '"\n')
+echo "LINUX_ID = ${LINUX_ID}"
+LINUX_CODENAME=$(grep ^VERSION_CODENAME= $OS_RELEASE_LOCATION | sed 's/^VERSION_CODENAME=//' | tr -d '"\n')
+echo "LINUX_CODENAME = ${LINUX_CODENAME}"
+LINUX_VERSION_ID=$(grep ^VERSION_ID= $OS_RELEASE_LOCATION | sed 's/^VERSION_ID=//' | tr -d '"\n')
+echo "LINUX_VERSION_ID = ${LINUX_VERSION_ID}"
```

Comment:
FYI- there's a tool `lsb_release` that technically is supposed to report all the info in `/etc/lsb-release`/`/etc/os-release` which should provide all this information. I didn't use it before because I've had little success with figuring it out myself, but it's probably a better mechanism than parsing the files ourselves. $0.02 in case you wanted to pursue it, but not blocking.